fastchat/serve/monitor/topic_clustering.py

```diff
@@ -89,6 +92,21 @@ def get_embeddings(texts, model_name, batch_size):
     return embeddings.cpu()
 
 
+# Support OpenAI Embedding
+def get_openai_embeddings(texts, model_name, batch_size):
+    client = OpenAI()
+    texts = texts.tolist()
+
+    embeddings = []
+    for i in tqdm(range(0, len(texts), batch_size)):
+        text = texts[i : i + batch_size]
```

Comment:
how's batch size affect total runtime?

Reply:
If you don't use batch size and just do a normal loop it will take several hours. With a batch size of 256, 54K prompts takes only around 3-4 minutes.